opencti-platform/opencti-front/src/private/components/entities/organizations/Root.jsx

```diff
@@ -131,7 +131,7 @@ class RootOrganization extends Component {
             if (props) {
               if (props.organization) {
                 const { organization } = props;
-                const paddingRight = getPaddingRight(location.pathname, organization.id, '/dashboard/entities/organizations');
+                const paddingRight = viewAs === 'knowledge' ? getPaddingRight(location.pathname, organization.id, '/dashboard/entities/organizations') : '';
```

Comment:
We still need the padding right when in content tab, both in knowledge and author view. It needs to be changed to something like :
`const paddingRight = getPaddingRight(location.pathname, organization.id, '/dashboard/entities/organizations', viewAs === 'knowledge');`